arangod/Aql/ExecutionNode.cpp

```diff
@@ -2171,33 +2168,7 @@ bool SubqueryNode::mayAccessCollections() {
 std::unique_ptr<ExecutionBlock> SubqueryNode::createBlock(
     ExecutionEngine& engine,
     std::unordered_map<ExecutionNode*, ExecutionBlock*> const& cache) const {
-  auto const it = cache.find(getSubquery());
-  TRI_ASSERT(it != cache.end());
-  auto subquery = it->second;
-  TRI_ASSERT(subquery != nullptr);
-
-  ExecutionNode const* previousNode = getFirstDependency();
-  TRI_ASSERT(previousNode != nullptr);
-
-  auto outputRegisters = RegIdSet{};
-
-  auto outVar = getRegisterPlan()->varInfo.find(_outVariable->id);
-  TRI_ASSERT(outVar != getRegisterPlan()->varInfo.end());
-  RegisterId outReg = outVar->second.registerId;
-  outputRegisters.emplace(outReg);
-
-  auto registerInfos = createRegisterInfos({}, std::move(outputRegisters));
-
-  // The const_cast has been taken from previous implementation.
-  auto executorInfos =
-      SubqueryExecutorInfos(*subquery, engine.getQuery(), outReg, const_cast<SubqueryNode*>(this)->isConst());
-  if (isModificationNode()) {
-    return std::make_unique<ExecutionBlockImpl<SubqueryExecutor<true>>>(
-        &engine, this, std::move(registerInfos), std::move(executorInfos));
-  } else {
-    return std::make_unique<ExecutionBlockImpl<SubqueryExecutor<false>>>(
-        &engine, this, std::move(registerInfos), std::move(executorInfos));
-  }
+  THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_INTERNAL, "cannot instantiate SubqueryExecutor");
```

Comment:
Should we also add a `TRI_ASSERT(false)`?

Reply:
Ok, done